nix/checks.nix

```diff
@@ -22,399 +22,402 @@
     in
     {
       checks =
-        let
-          # Create a testing harness for a PostgreSQL package. This is used for
-          # 'nix flake check', and works with any PostgreSQL package you hand it.
-          # deadnix: skip
-          makeCheckHarness =
-            pgpkg:
-            let
-              pg_prove = pkgs.perlPackages.TAPParserSourceHandlerpgTAP;
-              inherit (self'.packages) pg_regress;
-              getkey-script = pkgs.stdenv.mkDerivation {
-                name = "pgsodium-getkey";
-                buildCommand = ''
-                  mkdir -p $out/bin
-                  cat > $out/bin/pgsodium-getkey << 'EOF'
-                  #!${pkgs.bash}/bin/bash
-                  set -euo pipefail
-
-                  TMPDIR_BASE=$(mktemp -d)
-
-                  KEY_DIR="''${PGSODIUM_KEY_DIR:-$TMPDIR_BASE/pgsodium}"
-                  KEY_FILE="$KEY_DIR/pgsodium.key"
-
-                  if ! mkdir -p "$KEY_DIR" 2>/dev/null; then
-                    echo "Error: Could not create key directory $KEY_DIR" >&2
-                    exit 1
-                  fi
-                  chmod 1777 "$KEY_DIR"
-
-                  if [[ ! -f "$KEY_FILE" ]]; then
-                    if ! (dd if=/dev/urandom bs=32 count=1 2>/dev/null | od -A n -t x1 | tr -d ' \n' > "$KEY_FILE"); then
-                      if ! (openssl rand -hex 32 > "$KEY_FILE"); then
-                        echo "00000000000000000000000000000000" > "$KEY_FILE"
-                        echo "Warning: Using fallback key" >&2
+        if system == "aarch64-linux" then
+          { }
+        else
```

Comment:
Yeah, it's a bit of a hack ... the idea is indeed to revert it!

Reply:
@yvan-sraka totally understood thanks for giving us a temporary way to do it!